tests/unit/model/test_datastore_version.py

```diff
@@ -144,7 +149,7 @@ def test_draft_version_release_pending():
     version = draft_version.version
     updates, update_type = draft_version.release_pending()
     assert update_type == "MINOR"
-    assert [update.dict() for update in updates] == [
+    assert [update.model_dump() for update in updates] == [
```

Comment:
Let's use the aliases, and exclude nones here too for the sake of consistency 🙏🏻

here it will break because of this change but if we change the release_status to releaseStatus it should be fine 👍🏻